website/docs/reference/warehouse-profiles/bigquery-profile.md

```diff
@@ -215,6 +215,51 @@ Database Error in model debug_table (models/debug_table.sql)
   compiled SQL at target/run/bq_project/models/debug_table.sql
 ```
 
+### Service Account Impersonation
+You may want your models to be built using a dedicated service account that has
```

Comment:
Could you also add `<Changelog>New in v0.18.0</Changelog>` here, and link this page in the "New and changed documentation" section of the [0-18-0 migration guide](https://github.com/fishtown-analytics/docs.getdbt.com/blob/next/website/docs/docs/guides/migration-guide/upgrading-to-0-18-0.md)?